pyproject.toml

```diff
@@ -24,19 +24,9 @@ networkx = "^2.8.4"
 commandlib = "^0.3.5"
 rbf = {git = "https://github.com/frthjf/RBF.git", rev = "master"}
 neuroh5 = {git = "https://github.com/iraikov/neuroh5.git"}
-Sphinx = {version = "^4.5.0", optional = true, extras = ["docs"]}
-pydata-sphinx-theme = {version = "0.9.0", optional = true, extras = ["docs"]}
-readthedocs-sphinx-search = {version = "^0.1.2", optional = true, extras = ["docs"]}
-sphinx-autodoc-typehints = {version = "^1.18.1", optional = true, extras = ["docs"]}
-myst-parser = {version = "^0.17.2", optional = true, extras = ["docs"]}
-numpydoc = {version = "^1.3.1", optional = true, extras = ["docs"]}
-sphinx-togglebutton = {version = "^0.3.1", optional = true, extras = ["docs"]}
-sphinx-copybutton = {version = "^0.5.0", optional = true, extras = ["docs"]}
-myst-nb = {version = "^0.15.0", optional = true, extras = ["docs"]}
-sphinx-click = {version = "^4.3.0", optional = true, extras = ["docs"]}
-sphinxcontrib-mermaid = {version = "^0.7.1", optional = true, extras = ["docs"]}
```

Comment:
Can we revert this block of code? We need it for documentation build in ReadTheDocs.

Reply:
Yes, I will try to add it back. Poetry was getting stuck computing some sphinx-related dependencies, but I will try adding them back and regenerating the lock file.

Reply:
> Can we revert this block of code? We need it for documentation build in ReadTheDocs.

Actually, why are the documentation tools dependencies? Is there no separate category for documentation dependencies? Thanks for clarifying.

Reply:
Yes, that was the purpose for `extras = ["docs"]`. Although, I'll double check the poetry, I think they changed the syntax a little for optional dependencies.

Reply:
I checked the `poetry` documentation for any update, and it looks like we still need to specify dependencies here with `optional=true` tag, and group them in `[tool.poetry.extras]` section (we already did below). Looks like `extras=["docs"]` tag is not needed anymore.

Reply:
Ok, thank you for checking. I am going to add them back.

Reply:
Unfortunately, poetry gets stuck resolving dependencies when I add this block back. I am think I am going to try adding them one by one. What is the poetry command line to add dependencies with optional=true?

Reply:
I think `poetry add ... -E docs -G dev` [(reference)](https://python-poetry.org/docs/cli/#options-4)

Reply:
Thanks @frthjf I ended up using poetry add --optional and it turned out there is a conflict between myst and particular versions of sphinx, so I added version constraints that don't send poetry into tailspin. @skim0119 I would appreciate if you take a look and see if the doc can build ok.

Reply:
You can check the build here:
https://miv-simulator.readthedocs.io/en/feature-pyrho/